src/integration/calculator.integration.test.js

```diff
@@ -0,0 +1,17 @@
+const request = require("supertest");
+const app = require("../../index");
+
+jest.mock("../mapper/calculator", () => {
+  return () => 30;
+});
+
+describe("Integration testcase for Calculator", () => {
+//   jest.setTimeout(10000);
```

Comment:
please remove commented lines if not needed

Reply:
Done